data/pagure_owner_alias.json

```diff
@@ -1038,12 +1037,11 @@
             "torsava"
         ], 
         "python3-ecosystem": [
-            "cstratak", 
             "hhorak", 
-            "mhlavink", 
             "pviktori", 
-            "asamalik", 
-            "torsava"
+            "torsava", 
+            "mhlavink", 
+            "asamalik"
```

Comment:
As a side note, I wonder what changes at https://src.fedoraproject.org/modules/python3-ecosystem and why :confused: